### ISSUE TITLE
tensorflow_datasets required as invisible dependency with 1.6.0

### ISSUE THREAD

**Issue by SleepySpaceBear** [openvinotoolkit/datumaro#1472]
When using datum version 1.6.0 with Python 3.10.12, I came across an attribute error when running the datum command. By editing import_util.py to see what was being passed to the function, I found that tensorflow_datasets is passed. Since I did not have tensorflow_datasets installed in my virtual environment, the program crashes. I resolved the issue by installing tensorflow_datasets, but tensorflow_datasets is not in the requirement files
```
$ datum 
Traceback (most recent call last):
  File "/home/gabriel/.virtualenvs/traffic/bin/datum", line 5, in <module>
    from datumaro.cli.__main__ import main
  File "/home/gabriel/.virtualenvs/traffic/lib/python3.10/site-packages/datumaro/cli/__main__.py", line 11, in <module>
    from ..util.telemetry_utils import (
  File "/home/gabriel/.virtualenvs/traffic/lib/python3.10/site-packages/datumaro/util/telemetry_utils.py", line 12, in <module>
    from datumaro.cli import commands, contexts
  File "/home/gabriel/.virtualenvs/traffic/lib/python3.10/site-packages/datumaro/cli/commands/__init__.py", line 7, in <module>
    from . import (
  File "/home/gabriel/.virtualenvs/traffic/lib/python3.10/site-packages/datumaro/cli/commands/download.py", line 14, in <module>
    from datumaro.components.extractor_tfds import (
  File "/home/gabriel/.virtualenvs/traffic/lib/python3.10/site-packages/datumaro/components/extractor_tfds.py", line 49, in <module>
    tfds = lazy_import("tensorflow_datasets")
  File "/home/gabriel/.virtualenvs/traffic/lib/python3.10/site-packages/datumaro/util/import_util.py", line 13, in lazy_import
    loader = importlib.util.LazyLoader(spec.loader)
AttributeError: 'NoneType' object has no attribute 'loader'
```

**Comment by SleepySpaceBear** [comment]
If I have time this week, I'm willing to work on a PR and to figure out whether tensorflow_datasets is actually required or not. If it is, I'll just update the requirement files. If not, I'll figure out something else

**Comment by sooahleex** [comment]
Hi @SleepySpaceBear, Thank you for your interest in our project. This bug has been already reported in #1444 and has been fixed through #1454. It will be included in the upcoming release, version 1.6.1. Please retry with version 1.6.1 and if you still encounter the issue, please let me know.
